### DISCUSSION TITLE
How to use backticks instead of double quotes for column names

### DISCUSSION THREAD

**Discussion by bsandell** [dbeaver/dbeaver#20419] [Q&A]
I set up a custom jdbc driver in DBeaver.  When I generate a select query, the column names and table name are surrounded in double quotes.  My back end expects backticks instead of double quotes.  Is there a way that I can configure this connection/driver to use backticks instead of double quotes?

**Comment by LonwoLonwo** [comment]
Hello @bsandell 

Maybe, changing this parameter for your driver will help you:

![2023-07-03 12_07_59-DBeaver 23 1 1 - _default 1_ Script-21](https://github.com/dbeaver/dbeaver/assets/45152336/ac4c6262-d8b8-43d4-97f9-ed4f64becc44)

**Reply by bsandell** [reply]
Thanks for this suggestion!  Unfortunately, it didn't help (unless I didn't use it correctly).  I changed the value to `:` (that's backtick:backtick) but it still double quotes all of the identifiers.
